### ISSUE TITLE
Bootstrap 4 support

### ISSUE THREAD

**Issue by getvivekv** [vitalets/x-editable#1159]
Hi
Do you plan to introduce support for Bootstrap 4?

**Comment by webceoboy** [comment]
++1

**Comment by Brady1992** [comment]
同问。

**Comment by Brady1992** [comment]
+1

**Comment by hlrsky** [comment]
++1

**Comment by haggy** [comment]
+1 but I don't think it's going to happen. The status in the README is not-maintained and last commit over a year ago.

Im not a javascript/frontend wiz but I would hope the maintainer would consider releasing this to someone that wants to take it over because it is pretty nice 😄

**Comment by getvivekv** [comment]
@haggy  This is an open-source project and hosted on github. Anybody can fork it and make a modification. What do you mean by "release this"?

**Comment by haggy** [comment]
@getvivekv Yes anyone can fork it but it's better if the maintainer will openly admit they are no longer maintaining this project. That way the project can be handed over to someone else or the "main" fork can officially say they are the active version of the project. It's a matter of semantics but it helps keep the landscape cleaner.

**Comment by haggy** [comment]
It's also made more important by the fact that the Bootstrap 4 site still links directly to this repo.

**Comment by getvivekv** [comment]
@haggy  The developer clearly said this is not longer being maintained on their home page. Please take a look at it 
https://github.com/vitalets/x-editable

![image](https://user-images.githubusercontent.com/914305/81344081-1de74500-9084-11ea-9722-3301d2222ffb.png)

**Comment by haggy** [comment]
@getvivekv Hence my original comment:

> I would hope the maintainer would consider releasing this to someone

That means making someone else the official Maintainer in GH so that devs can submit PR's again. Right now __there are 40 open PR's__. 
Let me know if there is any other confusion on your side about this.

**Comment by VincentLanglet** [comment]
@haggy see https://github.com/vitalets/x-editable/issues/610

You can ask to maintain the project if you want to.

**Comment by webceoboy** [comment]
+1

**Comment by erycson** [comment]
BS4 Inline:

```javascript
$('.selector').editable()
    .on('shown', function (e, editable) {
      const input = editable.input.$tpl.attr('class', 'form-control form-control-sm');
      const container = input.parent().parent();
      container.find('.btn-default').removeClass('btn-default').addClass('btn-secondary');
      container.find('i.glyphicon-ok').removeClass('glyphicon glyphicon-ok').addClass('material-icons').text('done');
      container.find('i.glyphicon-remove').removeClass('glyphicon glyphicon-remove').addClass('material-icons').text('clear');
      container.find('.editable-clear-x').addClass('material-icons').css('font-size', '14px');
    })
```

**Comment by ilramdhan** [comment]
For Bootstrap 4 its work, check this [https://github.com/vitalets/x-editable/issues/1189#issue-2472436889](url)
